web_src/css/repo.css

```diff
@@ -66,6 +66,12 @@
   overflow-x: auto;
 }
 
+.issue-content-right .dropdown > .menu .item-secondary-info small {
```

Comment:
Could the style be on `.issue-content-right .dropdown > .menu .item-secondary-info` instead of `small`?